commercial/app/views/jobs/jobsBlended.scala.html

```diff
@@ -0,0 +1,14 @@
+@(job: model.commercial.jobs.Job, clickMacro: Option[String])
+
+<div class="advert__blended-container advert__blended-container--jobs">
+    <div class="advert__blended-container__title">
```

Comment:
I would probably rather see multiple objects:
`advert`
`advert-blended`
`advert-something-else`

It is easier to reason about. So if I want to work on advert blended I will work in `advert-blended.scss` file. It is easy to follow.